src/main/java/org/folio/search/model/metadata/FieldDescription.java

```diff
@@ -24,4 +28,12 @@
    * Field type.
    */
   private FieldType type;
+
+  /**
+   * Pre-defined type for the field, if applicable.
+   */
+  @JsonProperty("$type")
+  @Getter(AccessLevel.PACKAGE)
+  @Setter(AccessLevel.PACKAGE)
```

Comment:
I think that using package level setters and getters in this model is not necessary

Reply:
Modified to be public